packages/design-system-react/src/components/AvatarGroup/AvatarGroup.tsx

```diff
@@ -1,22 +1,28 @@
-import React, { useCallback } from 'react';
+import React, { useCallback, forwardRef } from 'react';
```

Comment:
Import and other fixes that are enabled by removing this file from the ignore list